jib-build-plan/src/main/java/com/google/cloud/tools/jib/api/buildplan/FileEntry.java

```diff
@@ -121,12 +158,13 @@ public boolean equals(Object other) {
     FileEntry otherFileEntry = (FileEntry) other;
     return sourceFile.equals(otherFileEntry.sourceFile)
         && extractionPath.equals(otherFileEntry.extractionPath)
-        && Objects.equals(permissions, otherFileEntry.permissions)
-        && Objects.equals(modificationTime, otherFileEntry.modificationTime);
```

Comment:
Perhaps `Objects.equals` was necessary in the past when `permissions` and `modificationTime` were `@Nullable`. The `java.util.Objects.equals` is implemented like the following, so we don't need it.

```java
    public static boolean equals(Object a, Object b) {
        return (a == b) || (a != null && a.equals(b));
    }
```